Dockerfile

```diff
@@ -91,10 +91,10 @@ RUN if [ ! -f /tmp/resources/geoserver.zip ]; then \
     && rm -rf /tmp/geoserver
 
 # Install any plugin zip files in resources/plugins
-RUN if ls "/tmp/resources/plugins/*.zip" > /dev/null 2>&1; then \
```

Comment:
These quotes (for `ls` and `mv` but not for `for`) indicate a file with a literal asterisk in its name.